src/Resources/contao/dca/tl_settings.php

```diff
@@ -205,12 +205,6 @@
 				array('tl_settings', 'checkTemplateFiles')
 			)
 		),
-		'maxImageWidth' => array
```

Comment:
If we remove the field, how are people supposed to reset it without having to adjust the `localconfig.php` file?

Reply:
I think adjusting the `localconfig.php` should be OK. But we could also keep it in tl_settings and add “(deprecated)” to the Label.

Reply:
@contao/developers What do you think?

Reply:
Removing is fine. People are quite used to editing the `localconfig.php` anyway as there are other settings that have never been configurable in the UI. I'm okay with that change.